include/engine/guidance/assemble_steps.hpp

```diff
@@ -204,22 +205,24 @@ inline std::vector<RouteStep> assembleSteps(const datafacade::BaseDataFacade &fa
         // intersections contain the classes of exiting road
         intersection.classes = facade.GetClasses(facade.GetClassData(target_node_id));
         BOOST_ASSERT(duration >= 0);
-        steps.push_back(RouteStep{step_name_id,
-                                  facade.GetNameForID(step_name_id).to_string(),
-                                  facade.GetRefForID(step_name_id).to_string(),
-                                  facade.GetPronunciationForID(step_name_id).to_string(),
-                                  facade.GetDestinationsForID(step_name_id).to_string(),
-                                  facade.GetExitsForID(step_name_id).to_string(),
-                                  NO_ROTARY_NAME,
-                                  NO_ROTARY_NAME,
-                                  duration / 10.,
-                                  distance,
-                                  weight / weight_multiplier,
-                                  target_mode,
-                                  maneuver,
-                                  leg_geometry.FrontIndex(segment_index),
-                                  leg_geometry.BackIndex(segment_index) + 1,
-                                  {intersection}});
+        steps.push_back(
+            RouteStep{step_name_id,
+                      facade.GetNameForID(step_name_id).to_string(),
+                      facade.GetRefForID(step_name_id).to_string(),
+                      facade.GetPronunciationForID(step_name_id).to_string(),
+                      facade.GetDestinationsForID(step_name_id).to_string(),
+                      facade.GetExitsForID(step_name_id).to_string(),
+                      NO_ROTARY_NAME,
+                      NO_ROTARY_NAME,
+                      duration / 10.,
+                      distance,
+                      weight / weight_multiplier,
+                      target_mode,
+                      maneuver,
+                      leg_geometry.FrontIndex(segment_index),
+                      leg_geometry.BackIndex(segment_index) + 1,
+                      {intersection},
+                      facade.IsLeftHandDriving(target_node_id)}); // TODO: this one is wrong
```

Comment:
Why is it wrong?

Reply:
That comment is referring to itself (the comment is wrong) ;-)  Deleting.